added pause to keep ps window open after run

mircojung opened the pull request:
Hi,
for simpler use (right click -> execute) i added a pause in the end to keep the window open. 

Feel free to pull or discard :)

Mirco

mircojung committed:
added pause to keep ps window open after run

perryflynn commented:
Just putting a pause into the script is not really fancy for all which want to execute that script in a existing window. We could try this to check if the script was double clicked:

https://social.technet.microsoft.com/Forums/ie/en-US/5fc1df45-77f2-4922-8eec-325594266635/how-detect-script-is-executed-from-command-line-or-by-double-click?forum=winserverpowershell

```ps1
$commandLine = (Get-WmiObject Win32_Process -Filter "ProcessID=$pid").CommandLine
```

mircojung committed:
added switch to do not pause if wanted

Looks like it's quiet hard to determine if script is running by click or ps window. 
(run out of cmd.exe is resolving to parent = cmd.exe; out of powershell _or_ by click resolves to parent = explorer.exe)

So i added a "nopause" arg for running automated without someone have to press enter.

mircojung commented:
Well, you are right. That "nopause" parameter is not really beautiful, too , but works. 
Automatic detection looks quiet hard (see commit description)

perryflynn commented:
Implemented by 9bc13787c28c33eb1ba25a23170396ae70b89a1d.
